content/en/logs/troubleshooting/_index.md

```diff
@@ -19,19 +25,19 @@ Fix the issue at the source by checking which `service` and `source` are impacte
 
 ## Unable to parse timestamp key from JSON logs
 
-If you are unable to convert the timestamp of JSON logs to a [recognized date format][4] before they are ingested into Datadog, follow these steps to convert and map the timestamps using Datadog's [arithmetic processor][5] and [log date remapper][6]:
+If you are unable to convert the timestamp of JSON logs to a [recognized date format][6] before they are ingested into Datadog, follow these steps to convert and map the timestamps using Datadog's [arithmetic processor][5] and [log date remapper][6]:
 
-1. Navigate to the [Pipelines][7] page.
+1. Navigate to the [Pipelines][9] page.
 
 2. In **Pipelines**, hover over **Preprocessing for JSON logs**, and click the pencil icon.
 
 3. Remove `timestamp` from the reserved attribute mapping list. The attribute is not being parsed as the official timestamp of the log during preprocessing.
 
 {{< img src="logs/troubleshooting/preprocessing_json_timestamp.png" alt="The preprocessing for JSON logs configuration box with the date attributes, which includes timestamp by default" style="width:90%" >}}
 
-2. Set up the [arithmetic processor][5] so that the formula multiples your timestamp by 1000 to convert it to milliseconds. The formula's result is a new attribute.
+2. Set up the [arithmetic processor][7] so that the formula multiples your timestamp by 1000 to convert it to milliseconds. The formula's result is a new attribute.
```

Comment:
```suggestion
2. Set up the [arithmetic processor][7] so that the formula multiplies your timestamp by 1000 to convert it to milliseconds. The formula's result is a new attribute.
```
I know this typo was already there, sorry we didn't catch it sooner!